### ISSUE TITLE
content type names/ids not handled correctly

### ISSUE THREAD

**Issue by janro1** [contentful-labs/cf-content-types-generator#11]
Hi,

if a content type has an id starting with a number or if the id contains dashes, like "3-0-Model" the type generation breaks.

**Comment by marcolink** [comment]
@janro1 good catch, will look into it.

**Comment by marcolink** [comment]
@janro1 
Ok, we currently don't generate validate module names for all cases. The `id` field is used to later generate a module name here: 
https://github.com/contentful-labs/cf-content-types-generator/blob/1f90fc8a979e60ca492b646e59844bce3df80de4/src/utils.ts#L3

I can think of a transform like this:

```js
expect(moduleName('helloWorld')).to.equal('HelloWorld');
expect(moduleName('hello-World')).to.equal('HelloWorld');
expect(moduleName('0helloWorld')).to.equal('HelloWorld0');
expect(moduleName('0-helloWorld')).to.equal('HelloWorld0');
expect(moduleName('0123-helloWorld')).to.equal('HelloWorld0123');
expect(moduleName('hello World')).to.equal('HelloWorld');
expect(moduleName('hello-World')).to.equal('HelloWorld');
```

Would that work for you?

**Comment by janro1** [comment]
Hi @marcolink, LGTM :)

**Comment by marcolink** [comment]
@janro1 provided a fix here #12
